dev/sparktestsupport/modules.py

```diff
@@ -693,6 +693,56 @@ def __hash__(self):
         "pyspark.pandas.tests.test_typedef",
         "pyspark.pandas.tests.test_utils",
         "pyspark.pandas.tests.test_window",
+        # unittests - Spark Connect parity tests
```

Comment:
I think we should move the tests into `pyspark-connect`, and then make `pyspark-connect` also depends on `pyspark_core`

Reply:
That makes pretty much sense to me.

Reply:
Updated. Thanks! :-)